### ISSUE TITLE
Autolinking for components/requirements that are defined in multiple places.

### ISSUE THREAD

**Issue by kgal** [commoncriteria/transforms#23]
I’m not sure if there’s a good way to automate this, but the automatic link parser is catching references to external PPs that shouldn’t be linked. For example, the EA for FCS_CKM.2(1) in the “modified from MDF PP” section has the following aactivity text: 
“For all key establishment schemes that conform to NIST SP 800-56A or 800-56B, refer to the evaluation activity for FCS_CKM.2(1) in the MDF PP.”
However, the FCS_CKM.2(1) link just points to the first instance of FCS_CKM.2 in the VPN Client Module, which is in the “Modified from GPOS PP” section. In reality we don’t want this linked at all because the aactivity text is referencing an SFR in a different PP.

**Comment by robertmclemons** [comment]
I think this will just have to be solved through educating people to use the ```<no-link>``` element.

**Comment by robertmclemons** [comment]
I agree with myself from last year. Maybe in the future we will be able to link between documents, but not yet.
